File: P17_0604/P17_0604(2).py




def printRepeating(arr,size) :
	count = [0] * size
	print(" Repeating elements are ",end = "")
	for i in range(0, size) :
		if(count[arr[i]] == 1) :
			print(arr[i], end = " ")
		else :
			count[arr[i]] = count[arr[i]] + 1


# Driver code
arr = [4, 2, 4, 5, 2, 3, 1]
arr_size = len(arr)
printRepeating(arr, arr_size)


		

File: P17_0604/P17_0604(2).cpp
#include<stdio.h>
#include<stdlib.h>

void printRepeating(int arr[], int size)
{
int *count = (int *)calloc(sizeof(int), (size - 2));
int i;

printf(" Repeating elements are ");
for(i = 0; i < size; i++)
{
	if(count[arr[i]] == 1)
	printf(" %d ", arr[i]);
	else
	count[arr[i]]++;
}
}	

int main()
{
int arr[] = {4, 2, 4, 5, 2, 3, 1};
int arr_size = sizeof(arr)/sizeof(arr[0]);
printRepeating(arr, arr_size);
getchar();
return 0;
}
